plugins/parsers/grok/parser.go

```diff
@@ -186,7 +186,10 @@ func (p *Parser) Compile() error {
 	p.loc, err = time.LoadLocation(p.Timezone)
 	if err != nil {
 		p.Log.Warnf("Improper timezone supplied (%s), setting loc to UTC", p.Timezone)
-		p.loc, _ = time.LoadLocation("UTC")
+		p.loc, err = time.LoadLocation("UTC")
+		if err != nil {
+			return err
+		}
```

Comment:
We probably should do
```suggestion
		p.loc = time.UTC
```
instead as UTC is defines as a variable in time's zoneinfo.